### DISCUSSION TITLE
Startseite funktioniert nicht mehr

### DISCUSSION THREAD

**Discussion by uiapoi** [FriendsOfREDAXO/.github#39] [Q&A] [answered]
Hallo zusammen,
Wer kann helfen?
Ich habe einen falschen Artikel angelegt, der nicht zu der Struktur passte. Jetzt baut sich  die Startseite nicht mehr auf. Da ich das nur übergangsweise mache, finde ich nichts, um die Startseite wiederherzustellen. 
Danke!!!

**Comment by eaCe** [comment]
@uiapoi das klingt nach einem Problem das ein wenig mehr Informationen benötigt. Komm doch gerne in den Slack-Kanal, dort kann dir eventuell schneller geholfen werden. https://redaxo.org/slack/

**Comment by dpf-dd** [comment]
Wie @eaCe richtig geschrieben hat: Wir brauchen mehr Infos.
Wenn "etwas nicht zur Struktur passt", dann kommt man dem mit dem Adminer-AddOn in der Regel ganz gut auf die Schliche.
Es könnte natürlich auch von Modulen oder Templates kommen, die auf der Startseite geladen werden. Es könnte auch ein PHP 8.x-"Problem" sein... Warten wir einfach mal ab, ob sich @uiapoi dazu noch präziser äußern kann... :)

**Comment by tyrant88** [comment]
Klingt für mich erstmal eher nach einfachen Bedienungs-Verständnis-Problem. Vielleicht das History-PlugIn verwenden, wenn installiert.

**Comment by uiapoi** [comment]
Hi Leute,

Danke für das Feedback. Die Startseite versucgt auf gelöschte tabinhalte zuzugreifen. Irgendwie wurden die nicht richtig verarbeitet.  Cache löschen. History Plugin gibt es leider nicht

**Comment by uiapoi** [comment]
Sorry, Cache löschen???

**Reply by eaCe** [reply]
In Slack geht das wirklich einfacher :D

**Comment by tyrant88** [comment]
History PlugIn ist das hier:
![Zwischenablage-1](https://user-images.githubusercontent.com/1277494/234923650-a604a9e7-5fcb-4a87-9629-e7d0af4ecc22.jpg)

**Comment by uiapoi** [comment, marked as answer]
Danke, habe es mit einem befreundeten ITler vorerst hingekriegt. Deshalb keine Zeit für Slack Einrichtung gehabt. Hole ich nach für weitere Probleme. Schönen Abend!

**Reply by eaCe** [reply]
Schön das es doch funktioniert hat.
